### ISSUE TITLE
Documentation in Module twisted.words._version

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6480]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Upasana reported|
|-|-|
|Trac ID|trac#6480|
|Type|enhancement|
|Created|2013-04-26 19:47:00Z|
|Branch|https://github.com/twisted/twisted/tree/version-header-6480|

************* Module twisted.words._version
W9001:  1,0: Missing copyright header
W9208:  1,0: Missing docstring

Attachments:

* [bug_6480.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b04/b04ba543fa77efa5bdae91e588e46f20481be3b9/f4ad5dfd5b532f1a199cb43b5cbaebf0426a9b73.patch) (611 bytes) - added by Upasana on 2013-04-26 19:49:15Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6480 6480
type__enhancement enhancement
reporter__Upasana Upasana
priority__low low
milestone__ 
branch__branches_version_header_6480 branches/version-header-6480
branch_author__Upasana Upasana
status__closed closed
resolution__fixed fixed
component__words words
keywords__easy easy
time__1367005620000000 1367005620000000
changetime__1368551223000000 1368551223000000
version__None None
owner__thijs thijs
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|Upasana commented|
|-|-|

twisted.words._version is an auto-generated file, generated by twisted.python._release.generateVersionFileData function, so I need to amend twisted/python/_release.py, not twisted.words._version

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Upasana removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Upasana set owner to Upasana|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Upasana's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Upasana removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_8"></a>|@keturn set owner to @keturn<br>@keturn set status to `assigned`|
|-|-|

assigning for review

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra commented|
|-|-|

(In [38402]) Branching to 'version-header-6480'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra commented|
|-|-|

(In [38403]) apply bug_6480.patch, refs [#6480](https://github.com/twisted/twisted/issues/6480)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_11"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `new`|
|-|-|

Thanks for your patch. Forced a [build](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/version-header-6480), will merge when it's green.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_12"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38408]) Merge version-header-6480: Add copyright header and docstring in the auto-generated _version module for subprojects.

Author: Upasana
Reviewer: thijs
Fixes: [#6480](https://github.com/twisted/twisted/issues/6480)
